packages/beeq/src/components/card/_storybook/bq-card.stories.tsx

```diff
@@ -0,0 +1,99 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit-html';
+
+import mdx from './bq-card.mdx';
+import { CARD_TYPE } from '../bq-card.types';
+
+const meta: Meta = {
+  title: 'Components/Card',
+  component: 'bq-card',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    type: { control: 'select', options: [...CARD_TYPE] },
+  },
+  args: {
+    type: 'default',
+  },
+};
+export default meta;
+
+type Story = StoryObj;
+
+const Template = (args: Args) => html`
+  <div class="flex flex-row gap-m items-end">
+    <bq-card type=${args.type}>
```

Comment:
Always use [ifDefined directive from lit-html](https://lit.dev/docs/templates/directives/#ifdefined) for attribute/properties that aren't boolean.